File: retos-python/23_numeros_perdidos.py
# Resultado 4 ⭐
def find_missing_numbers(nums):
    numMax = max(nums)
    values = [i + 1 for i in range(numMax)]

    c1 = set(nums)
    c2 = set(values)

    return list(c2.difference(c1))

File: retos-ts/23_numeros_perdidos.ts
// 5 ⭐ 
function findMissingNumbers(nums: number[]): number[] {
    const limit:number = Math.max(...nums);
    let values = new Array(limit).fill(0).map((_,i) => i + 1)    
    
    const c1 = new Set(nums)
    const c2 = new Set(values)

    return [...c2.difference(c1)]
}